decidim-core/app/controllers/decidim/profiles_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Decidim
+  # The controller to handle the user's public profile page.
+  class ProfilesController < Decidim::ApplicationController
+    skip_authorization_check
+
+    helper Decidim::Messaging::ConversationHelper
+
+    helper_method :user
+
+    def show; end
+
+    private
+
+    def user
+      @user ||= Decidim::User.find_by!(nickname: params[:nickname])
```

Comment:
Should we scope this to the current organization?

Reply:
Absolutely! The nickname is not even unique accross organizations! I'll update!